### PR TITLE
Add Log Forwarding

### LINKED ISSUE #1: Logging Passthrough
Howdy! Was just build my own integration when I stumbled across this, and it looks awesome! About the only thing missing in logging integration. Shouldn't be to hard to add a log appender if you'd like the contribution?

### PR DESCRIPTION
**Pull request by RichiCoder1**
Adds the ability to send `aurelia-logging` messages to app insights.

Fixes #1;

Sorry bout the package.json whitespace change. Not sure what happened there.

### CODE REVIEW

**Comment by miguelzakharia** [comment]
Sorry that I have not been able to review this yet, I've had a very busy week. If I don't get to it today then I will certainly complete it this weekend.

**Comment by RichiCoder1** [comment]
No worries :). As the beleagured maintainer of some of my own FOSS stuff, I understand

File: src/logAppender.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	AppInsights
+} from "applicationinsights-js";
+
+export default class AppInsightsAppender {
+	constructor(plugin) {
+		this._plugin = plugin;
+	}
+
+	debug(logger, message) {
+		if (!this._plugin.initialized) {
```
Comment (miguelzakharia) [inline src/logAppender.js]: Line 136 in `application-insights` already checks if the plugin has been initialized. I don't think the checks in `logAppender` are necessary and `logAppender` probably shouldn't care about the plugin anyway, right?

**Review by miguelzakharia — CHANGES_REQUESTED** [review]

File: src/logAppender.js (hunk 1 above)
Comment (RichiCoder1) [reply src/logAppender.js]: Ohh, good catch. It wasn't originally structured like that.

**Comment by RichiCoder1** [comment]
Addressed comments and corrected a bad assumption. Should be good to go.

**Review by miguelzakharia — APPROVED** [review]

**Pull request merged**